prime-router/docs/playbooks/add_ip_blacklist.md

```diff
@@ -0,0 +1,54 @@
+# Add IP to blacklist
+
+## Introduction
+There may be a time where we need to blacklist a specific IP address due to a bad actor or misconfiguration from a sender/reciever. After checking into the various options, it seems that blocking the user at the Azure WAF/Front Door will be easier to deploy and remove than through Akamai. Please refer to the following steps.
+
+
+## Add IP to Blacklist Steps
+### Console
+1. Log into the Azure Portal.
+2. Click the resource group you want to add the blacklist to.
+3. In the main resource group window, find the Front Door WAF Policy and click on it.
+4. Inside the WAF Policy Screen, click on Custom Rule under the Settings section of the menu.
+5. Next click on Add Custom Rule.
+6. Provide the followiing in the opened menu
+    - Name: Rule of the name you want to create.
+    - Priority: Priority number based on other Rules.
+    - Conditions
+        - Match Type: IP Address
+        - Match Variable: RemoteAddr
+        - Operation: Does Contain
+        - IP Address or Range: *IP YOU WISH TO BLOCK*
+    - Then
+      - Deny Traffic
+
+### Amazon Cli
+1. Add a custom WAF Rule with the following changes: 
+- **IPAllowPolicyExampleCLI**: The Policy name for the WAF Policy
```

Comment:
Does this policy already exist? Or will it get auto-created for you when this first command is run?

Also, can you mention that "IPAllowListRule" isn't what you should call the rule when you run it, it's just a placeholder for the name for this particular rule? (At least I think that's correct.)